MAUI/Calendar/getting-started.md

```diff
@@ -130,11 +130,11 @@ this.Calendar.MonthView.FirstDayOfWeek = DayOfWeek.Monday;
 
 ## Date selection
 
-The Calendar allows the user to select a single date, multiple dates, or a range of dates by interaction or programmatic selection. The default selection mode is `Single`.
+Calendar allows the user to Select a single date, multiple dates, or a range of dates through interaction and programmatically. The default selection mode is `Default`. 
 
-The selection details can be obtained by using the SelectionChanged event of the calendar. It returns `CalendarSelectionChangedEventArgs`, which holds the details about the selected date or range.
+The selection details can be obtained by using SelectionChanged event of calendar. It returns `CalendarSelectionChangedEventArgs` which holds the details about selected date or range.
```

Comment:
The selection details can be obtained by using the SelectionChanged event of the calendar. It returns `CalendarSelectionChangedEventArgs,` which holds the details about the selected date or range.

Reply:
Content changed